dagmc/dagnav.py

```diff
@@ -111,6 +111,31 @@ def write_file(self, filename):
         """
         self.mb.write_file(filename)
 
+    def add_groups(self, group_map):
+        """Adds groups of DAGSets to the model.
+
+        Parameters
+        ----------
+        group_map : dict
+            A dictionary whose keys are 2-tuples of (str, int) containing the
+            group name and group ID respectively and whose values are iterables
+            of DAGSet objects or DAGSet ID numbers.
+        """
+        for (group_name, group_id), dagsets in group_map.items():
+            group = Group.create(self, name=group_name, group_id=group_id)
+
+            for dagset in dagsets:
+                if isinstance(dagset, DAGSet):
```

Comment:
This case isn't tested. No reason to think it won't work, but may as well keep the test coverage up...

Reply:
good point - I added a case for volume and surfaces